test/make.sh

```diff
@@ -50,6 +54,11 @@ compat_xargs() {
     compat=("--no-run-if-empty")
   fi
   xargs "${compat[@]}" "$@"
+  rval="$?"
+  if [[ -z "${NOWARN:-}" ]] && [[ "${rval}" -gt 0 ]]; then
+    echo "Warning: compat_xargs $* failed with exit code ${rval}" >&2
```

Comment:
We added this warning while debugging mysterious lint failures which were fairly silent.  Seemed good to keep.  The warning may be disabled, e.g. when using grep to match trailing whitespace.